### DISCUSSION TITLE
What are the *_id_pudl fields for?

### DISCUSSION THREAD

**Discussion by JesseWeinstein** [catalyst-cooperative/pudl#2290] [Help Me!] [answered]

**Comment by JesseWeinstein** [comment, marked as answer]
@bendnorman answered (on the OpenStreetMapUS Slack, where I originally asked this):

> The ***_id_pudl**  fields connect the EIA and FERC data. EIA and FERC do not share common keys for plants and utilities so each year we create PUDL ids for new utilities and plants and map the ***_id_eia** to the ***_id_ferc**.

**Reply by zaneselvans** [reply]
There is also `unit_id_pudl` which is a generation unit ID that we've assigned -- basically any ID that we're assigning to structure the data, which is **not** coming from some external source we've tried to suffix with `_id_pudl` so others know it's specific to our DB.
